packages/app/src/app/app.config.ts

```diff
@@ -1,10 +1,25 @@
 import { provideHttpClient, withFetch, withInterceptors } from '@angular/common/http';
-import { ApplicationConfig, provideZonelessChangeDetection } from '@angular/core';
+import { ApplicationConfig, ErrorHandler, provideZonelessChangeDetection } from '@angular/core';
 import { provideAnimationsAsync } from '@angular/platform-browser/animations/async';
 import { provideRouter, withComponentInputBinding } from '@angular/router';
 import { routes } from './app.routes';
 import { authInterceptor } from './auth/auth-http-interceptor';
 import { MAT_FORM_FIELD_DEFAULT_OPTIONS } from '@angular/material/form-field';
+import * as Sentry from '@sentry/angular';
+import { environment } from '../environments/environment';
+
+const enableSentry = Boolean(environment.sentryDsn);
+
+if (enableSentry) {
+  console.info('Enabling Sentry with DSN:', environment.sentryDsn);
+  Sentry.init({
+    dsn: environment.sentryDsn,
+    sendDefaultPii: false,
```

Comment:
I have this true in mine - thoughts?

Reply:
I have set to false to align for now - good to be cautious

Reply:
ya, I just am being cautious in case we're not allowed to collect IP addresses, not certain what the rules are